app/src/main/res/layout/image_region_selection_test_fragment.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:orientation="vertical">
+
+  <FrameLayout
+    android:id="@+id/image_parent_view"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    tools:context=".testing.ImageRegionSelectionTestActivity">
+
+    <org.oppia.app.player.state.ImageRegionSelectionInteractionView
+      android:id="@+id/clickable_image_view"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:adjustViewBounds="true"
+      android:src="@drawable/testing_fraction" />
+
+    <View
+      android:id="@+id/default_selected_region"
+      android:layout_width="24dp"
+      android:layout_height="24dp" />
+  </FrameLayout>
+
+  <Button
+    android:id="@+id/submit_button"
```

Comment:
As we discussed, the notion of enabling/disabling the submit button is technically unrelated to the image region selection. We should be testing when the region callback happens & what data is passed to it, not the outcome (whether a view is enabled).